client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -818,7 +818,7 @@ class CancelPurchaseForm extends React.Component {
 		const cancel = {
 			action: 'cancel',
 			disabled,
-			label: translate( 'Cancel Now' ),
+			label: translate( 'Confirm' ),
```

Comment:
I think this change makes sense when disabling auto-renew.  But when a refundable subscription is being cancelled and removed right away, the original text seems better since it's more clear about what will happen.

**Before:**

![cancel-refund-survey-before](https://user-images.githubusercontent.com/235183/126824637-bdef0c57-d4ac-4ec8-9cc4-fe9c49685727.png)

**After:**

![cancel-refund-survey-after](https://user-images.githubusercontent.com/235183/126824642-b5445165-de14-47a0-ad19-7e647545acbb.png)